functionalTest/test/pageobjects/page.js

```diff
@@ -0,0 +1,14 @@
+/**
+* main page object containing all methods, selectors and functionality
+* that is shared across all page objects
+*/
+module.exports = class Page {
+    /**
+    * Opens a sub page of the page
+    * @param path path of the sub page (e.g. /path/to/page.html)
+    */
+    open () {
+        return browser.url('/')
```

Comment:
Change this to '/index.html' and you'll be able to run the tests against the staging site

Reply:
Changes are done and pushed to GitHub.